src/main/java/it/pagopa/interop/probing/eservice/operations/model/view/EserviceView.java

```diff
@@ -77,4 +83,20 @@ public class EserviceView implements Serializable {
   @Column(name = "polling_frequency")
   private Integer pollingFrequency;
 
+  @Column(name = "polling_start_time")
+  private OffsetTime pollingStartTime;
+
+  @Column(name = "polling_end_time")
+  private OffsetTime pollingEndTime;
+
+  @NotNull
```

Comment:
If it is NotNull then it's better to use a default value and to create a constraint on DB column to allow only a domain values in case of inserting by DML.

Reply:
we cannot set a default value here because the related business logic is placed in another service.
The `NOT NULL` constraint in the table definition is enough to be sure that this column is always populated.

The enumeration constraint might be a good improvement, but let's open it as enhancement task and close this PR first